charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -260,6 +260,9 @@ ingress:
   #    hosts:
   #      - oidc-discovery.example.org
 
+  ## @param ingress.controllerType Specify what type of ingress controller your using. If generic, no additional annotations will be added to your ingresses. Must be one of [generic, ingress-nginx].
+  controllerType: generic
+
```

Comment:
If this is just about annotations i think we should link it somehow to the regular `annotations` section. `standardAnnotations` and `additionalAnnotations` or something like that. What do you think?

Reply:
For now its just annotations. It could easily involve non annotations in the future.

How would it go if it was standardAnnotations? The standardAnnotations vary by controllerType.

Reply:
Good point about future proofing. Lets keep it as is. Can we move it up under `className`? How about `other` instead of `generic`?

Reply:
Done and Done.